### PR TITLE
ARACHNE-2719  Update AD/LDAP user import routines to work with Authen…

### PR DESCRIPTION
**Pull request by YaroslavTir**
Fixes https://github.com/OHDSI/ArachneCentralAPI/issues/620
Update AD/LDAP user import routines to work with Authenticator
- upgrade version of authenticator

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -454,8 +454,8 @@
                 <configuration>
                     <imageName>${dockerImagePrefix}/${project.artifactId}</imageName>
                     <imageTags>
-                        <imageTag>${project.version}</imageTag>
-                        <imageTag>latest${latestSuffix}</imageTag>
+                        <imageTag>1.15.1-QA-PROXY</imageTag>
```
Comment (wivern) [inline pom.xml]: Hmm, this looks like some stub, doesn't?

File: src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java [hunk 2]
```diff
@@ -130,9 +131,9 @@
 
         String token = request.getHeader(tokenHeader);
         UserInfo userInfo = authenticator.refreshToken(token);
-        userService.findByUsername(userInfo.getUsername()).orElseThrow(() -> new AuthException("user not registered"));
+        userService.findByUsername(userInfo.getUser().getUsername()).orElseThrow(() -> new AuthException("user not registered"));
```
Comment (wivern) [inline src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java]: NPE might be thrown here

File: src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java [hunk 3]
```diff
@@ -130,9 +131,9 @@
...
 
-        return new JsonResult<>(JsonResult.ErrorCode.NO_ERROR, userInfo.getToken());
+        return new JsonResult<>(JsonResult.ErrorCode.NO_ERROR, userInfo.getAuthenticationInfo().getToken());
```
Comment (wivern) [inline src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java]: Same as above (NPE)

File: pom.xml (hunk 1 above)
Comment (YaroslavTir) [reply pom.xml]: fixed

File: src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java (hunk 2 above)
Comment (YaroslavTir) [reply src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java]: fixed

File: src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java (hunk 3 above)
Comment (YaroslavTir) [reply src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java]: fixed

**Pull request merged**